source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+Both Google and Apple require signatures of the application and push proxy to match. This means if you build the applications, you must host your own instance of MPNS.
+
+Building the apps can be an involved process. This requires that you have the skillset to maintain and deploy packaged apps. Part of this process includes, but is not limited to :
+
+- Obtaining/providing certificates for your custom application
+- Signing your custom applications
+- Distributing your applications via public or private app stores
+
+Your custom built apps are a fork of the Mattermost project. This means you will be responsible for adding in any feature or security updates.  
+
+To understand what’s involved, have your development team `read through our documentation <https://developers.mattermost.com/contribute/mobile/build-your-own/>`_.
+
+    **Note:** As of Mattermost 5.18, E20 customers can limit data sent to the HPNS. With this option a message containing only an ID. Once the mobile client receives this ID, the message contents are loaded from the server. Thus, the message contents are never transmitted through APNS/FCM.
+
+----
+
+#####################################################
+Mobile Deployment via Public App Stores (Recommended)
+#####################################################
+
+To proceed you must have a Mattermost Server installed and accessible. This is also true for  using the hosted version of the Mattermost push proxy (HPNS).
+
+*********************************
+Accessing the Mobile Applications
+*********************************
+
+The Mattermost mobile application is available for both Android and iOS devices. At this point, it’s as simple as having your users download the application and `point to your Mattermost Server URL <https://docs.mattermost.com/help/getting-started/signing-in.html#ios-setup>`_.
+
+    **Note:** The Mattermost Mobile apps are signed and have certificates associated with Mattermost and the public app stores. This means they will not work if you are privately hosting the Mattermost Push Proxy Service.
+
+`Mattermost for Android Devices <https://play.google.com/store/apps/details?id=com.mattermost.rn&hl=en_US>`_ (via Google Play)
+`Mattermost of iOS Devices <https://apps.apple.com/us/app/mattermost/id1257222717>`_ (via Apple App Store)
+
+If you do not desire (or require) additional security provide via an EMM Provider, your deployment is complete. Feel free to point your users to the `available documentation <https://docs.mattermost.com/guides/user.html>`_.
+
+The sub-sections below serve as a high-level guide to understanding this deployment model. When necessary, we point to the documentation for technical instruction.
+
+    **Note:** At a minimum, we recommend you follow our recommended steps to secure your deployment.
+
+********************************************
+Using an EMM Provider with Public Store Apps
```

Comment:
:more_pain:

Reply:
I think it too 40 of these for me to reach out to @enahum. But hey... we’ll figure it out. This is why we review.

Reply:
resolved in meeting